radiant/dags/init_iceberg_tables.py

```diff
@@ -1,44 +1,51 @@
 from airflow import DAG
 from airflow.decorators import task
 from airflow.utils.dates import days_ago
-from pyiceberg.catalog import load_catalog
-from pyiceberg.partitioning import PartitionField, PartitionSpec
-from pyiceberg.transforms import IdentityTransform
-
-from radiant.tasks.vcf.consequence import SCHEMA as CONSEQUENCE_SCHEMA
-from radiant.tasks.vcf.occurrence import SCHEMA as OCCURRENCE_SCHEMA
-from radiant.tasks.vcf.variant import SCHEMA as VARIANT_SCHEMA
 
 NAMESPACE = "radiant"
 
 default_args = {
-    "owner": "ferlab",
+    "owner": "radiant",
 }
 
 with DAG(
-    dag_id="init_iceberg_tables",
+    dag_id="radiant-init-iceberg-tables",
     default_args=default_args,
     start_date=days_ago(1),
     schedule_interval=None,
+    tags=["radiant", "iceberg"],
```

Comment:
Open question, should we always add the `NAMESPACE` as first tag ? 🤔 
I could do the same in all my dags as well.

Reply:
I dont think it's necessary. It will not change accross deployment.